### PR TITLE
Remove unnecessary words in the help tab #173

### LINKED ISSUE #173: Remove unnecessary words in the help tab
Description:
There are multiple instances of the comment # nolint scattered throughout the codebase. These comments are unnecessary and create clutter, making the code harder to read and maintain.

Task:

Locate and remove all occurrences of the # nolint comments from the code.
Ensure that removing these comments does not affect the functionality or the linting process of the project.
Expected Outcome:

The codebase is cleaner, with no # nolint comments present.
The readability and maintainability of the code are improved.
The functionality of the project remains intact, and the code still passes linting checks where applicable.

Screenshot 2024-04-30 at 12 55 02 PM

### PR DESCRIPTION
**Pull request by srepalli3**
Fixes #173

**What was changed?**

The unnecessary words " # nolint" has been removed from the general information tab. 
Screenshot 2024-09-16 at 4 59 30 PM

**Why was it changed?**

Deleting the words allows for better readability, it was necessary to delete the word since it has no place in the general information tab. 

**How was it changed?**

ui.R was updated so that the "#nolint" comment was removed from the actual text. 

**How was it tested**

Ran the MeltShiny.command to confirm that #nolint was removed from the text
Screenshot 2024-09-16 at 5 00 01 PM

### CODE REVIEW

File: code/.Rhistory [hunk 1]
```diff
@@ -0,0 +1,512 @@
+length_t_test <- t.test(parrots_data$Length_Live, parrots_data$Length_Playback, na.action = na.omit)
```
Comment (avazzano7) [inline code/.Rhistory:1]: This .Rhistory file may not need to be merged into the main branch

Comment (avazzano7) [inline code/.Rhistory:1]: this .Rhistory file may not need to be merged

**Review by avazzano7 — CHANGES_REQUESTED** [review]

File: code/.Rhistory (hunk 1 above)
Comment (kate-holdener) [reply code/.Rhistory:1]: looks like maybe this file should be added to .gitignore

Comment (BollepalliPranitha) [reply code/.Rhistory:1]: Yes , I will be adding the .Rhistory file to .gitignore

**Review by BollepalliPranitha — CHANGES_REQUESTED** [review]
The unnecessary word #nolint is removed from the Help page 
But when I tried uploading the data and after clicking on upload data 
I got this error

image

These are the input details
AYGU-1_MeltR.csv
Sample 1
No Blanks
Wavelength – 260
Nucleic acid type – RNA
Sequence – GACUAGUU
Molecular state - Homoduplex

**Comment by srepalli3** [comment]
> Merging can be performed automatically once the requested changes are addressed.

I fixed this by rebasing my branch with main.

**Review by BollepalliPranitha — APPROVED** [review]
Great work on cleaning up the code by removing the unnecessary # nolint comments! This change improves code quality and aligns with best practices by keeping the codebase clean and free from unnecessary tags.
The removal of the # nolint comments is appropriate, as it does not affect core logic.
There are no performance concerns associated with this change.

**Review by avazzano7 — APPROVED** [review]

File: code/.Rhistory (hunk 1 above)
Comment (kate-holdener) [reply code/.Rhistory:1]: @srepalli3 please remove .Rhistory file from this pull request. To do this, from your clone of the repo from the fix/issue_173 branch run:
git rm .Rhistory
git commit -m 'removed Rhistory'
git push

**Review by kate-holdener — CHANGES_REQUESTED** [review]
Please remove .Rhistory file from this PR.

File: code/.Rhistory (hunk 1 above)
Comment (srepalli3) [reply code/.Rhistory:1]: will do, thank you!

**Comment by BollepalliPranitha** [comment]
@srepalli3 The code has been thoroughly tested, and all functionalities are working as expected.

There were some merge conflicts with the fix/issue_173 branch. I have resolved the conflicts and successfully merged your code into that branch.

**Pull request merged**